src/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title>Title</title>
     <link href="style.css" rel="stylesheet">
-    <script type="text/javascript" src="../bower_components/jquery/src/jquery.js"></script>
+    <script src="scripts.js"></script>
```

Comment:
Die Datei liegt dann ja in `dist/scripts.js`, also im gleichen Verzeichnis wie die `*.html` Dateien. Wir müssen also keinen Pfad mehr angeben. Der Zugriff auf `../bower-components` war deshalb nicht möglich, weil Browser-Sync im `/dist/` gestartet wurde, und damit gar kein Zugriff auf eine höherliegende Datei möglich war:

![selection_002](https://cloud.githubusercontent.com/assets/229567/22361418/bc7631de-e459-11e6-82ad-bbdf716ab5d8.png)

Die Angabe von `type="text/javascript"` ist heutzutage eigentlich nicht mehr notwendig.